### PR TITLE
[risk=no] Refactor WorkspaceACL handling to avoid Java warning

### PR DESCRIPTION
**Pull request by calbach**
Slightly more automated parsing + gets rid of the unchecked Java warning.

Copied from my now deleted backfill script: https://github.com/all-of-us/workbench/pull/2221/files#diff-3e77476e30c3fda73b42c53d56c9f977R96

### CODE REVIEW

File: api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java [hunk 1]
```diff
@@ -134,19 +135,20 @@ public Workspace get(String ns, String firecloudName) {
...
-        fireCloudService.getWorkspaceAcl(workspaceNamespace, firecloudName);
-    Map<String, Object> aclsMap = (Map) firecloudWorkspaceAcls.getAcl();
-    Map<String, WorkspaceAccessEntry> userToAcl = new HashMap<>();
-    for (Map.Entry<String, Object> entry : aclsMap.entrySet()) {
-      WorkspaceAccessEntry acl =
-          new Gson().fromJson(entry.getValue().toString(), WorkspaceAccessEntry.class);
-      userToAcl.put(entry.getKey(), acl);
-    }
-    return userToAcl;
+    return extractAclResponse(fireCloudService.getWorkspaceAcl(workspaceNamespace, firecloudName));
```
Comment (jacmrob) [inline api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java]: looks good except why make this its own method?  seems like you could just put the above code into `getFirecloudWorkspaceAcls` to consolidate, and add a comment for clarity

**Comment by jacmrob** [comment]
seems like something went wrong in the return of that method; see the failed test.  i can give more context, but basically during clone when looking up the acls of the cloneFrom ws it is not finding anything, which is why it is returning a NO_ACCESS for each of the users who were on the ws (it thinks their acls were removed bc they are not found).   this is bc of how the method iterates; basically [this call](https://github.com/all-of-us/workbench/blob/master/api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java#L252) is returning empty which is then causing the code to go to the else clause.  so something from the changes you made are causing `getFirecloudWorkspaceAcls` to return null or something.

File: api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java (hunk 1 above)
Comment (calbach) [reply api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java]: Done.

**Comment by calbach** [comment]
The tests were not quite matching what is returned by Swagger codegen. It returns a Map, but more specifically a Gson TreeMap, which Gson can further parse. This is what my method was taking advantage of. I updated the tests to better match what happens on the server.

**Comment by calbach** [comment]
(I also ran the server locally to verify)

**Pull request merged**